CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Unreleased
 
 - On Windows, fix `WindowBuilder::with_maximized` being ignored.
+- On iOS, touch positions are now properly converted to physical pixels.
```

Comment:
Looks like breaking changes.

Reply:
I'm merely fixing a violation of our existing API guarantees. If anyone's relying on the current behavior, they're using the API incorrectly.